packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "salesforcedx-vscode-lwc",
-  "displayName": "LWC",
-  "description": "Provides code-editing features for LWC",
+  "displayName": "Lightning Web Components",
+  "description": "Provides code-editing features for Lightning web components.",
```

Comment:
The capitalization of this differs from the previous line. I assume that this is intentional but I just wanted to call it out to confirm.

Reply:
Yes, it's awkward, but accurate. We capitalize Lightning Web Components when we refer to the programming model (or it's in a title). We do not capitalize the web components part when we refer to actual components or creating them.